joerd/source/greatlakes.py

```diff
@@ -0,0 +1,171 @@
+from joerd.util import BoundingBox
+import joerd.download as download
+import joerd.check as check
+import joerd.srs as srs
+import joerd.mask as mask
+import joerd.tmpdir as tmpdir
+from joerd.mkdir_p import mkdir_p
+from shutil import copyfileobj
+import os.path
+import os
+import logging
+from osgeo import gdal
+import tarfile
+
+
+# hard-coded bounding boxes for each of the Great Lake datasets. this is because
+# there's no link between the name of each piece of data and its bounding box,
+# which was a design flaw / assumption for other data sources.
```

Comment:
In the future, would it be possible to push this into optional configuration for the data source?